manifests/base/deployment.yaml

```diff
@@ -113,6 +113,24 @@ spec:
                   name: jwt-secret
             - name: S3_BUCKET
               value: campaign-files
+            - name: IRIS_API_URL
+              value: https://developer.sandbox.irispay.bg/api/8
+            - name: BANK_BIC
+              value: UNCRBGSF
+            - name: PLATFORM_IBAN
+              value: test
+            - name: IMPORT_TRX_TASK_INTERVAL_MINUTES
+              value: '60'
+            - name: IRIS_AGENT_HASH
+              valueFrom:
+                secretKeyRef:
+                  name: iris-secret
+                  key: agent-hash
+            - name: IRIS_USER_HASH
```

Comment:
Can you confirm that both agent and user hash are supposed to be secrets?

Reply:
yes both are private secrets